LocaleEmulator/LocaleEmulator.h

```diff
@@ -346,7 +448,7 @@ OpenOrCreateLePeb(
     return LePeb;
 }
 
-#define ENABLE_LOG 1
+#define ENABLE_LOG 0
```

Comment:
Log closed.

Reply:
It‘s okay. Developers should know how to switch it back on.